soc/riscv/riscv-privilege/common/CMakeLists.txt

```diff
@@ -16,3 +16,9 @@ zephyr_sources(
 	idle.c
 )
 endif()
+
+if(!CONFIG_SOC_RISCV_TELINK_B93)
```

Comment:
Seems we already had a bug. Idle is not used for telink soc.
idle.c is inside our soc folder

Reply:
@mishadesh please fix issue as common B9X

Reply:
Fix added